app/features/atp/information/ui/idx/controller.ts

```diff
@@ -98,30 +100,40 @@ const infoValidationMiddleware = [
 
 // This is the root route and will redirect back to the appropriate gov.uk start page
 const getInfo = (req: Request, res: Response): void => {
-  if (!req.session.userData.basicInfo) {
-    req.session.userData.basicInfo = {};
+  if (!req.session.sessionData.identityEvidence) {
+    req.session.identityEvidence = [];
   }
-  const {
+
+  let informationAttributes;
+  const allIdentityEvidence = req.session.sessionData.identityEvidence;
+  if (allIdentityEvidence) {
+    informationAttributes = allIdentityEvidence.filter(filterInformation).slice(-1).map((evidence) => evidence.attributes)[0];
+  }
+
+  function filterInformation(allIdentityEvidence) {
+    return allIdentityEvidence.type == EvidenceType.BASIC_INFO;
+  }
```

Comment:
Filter information used before its defined? Again, this could be replaced with an arrow function